src/io/eval.rs

```diff
@@ -265,10 +327,16 @@ where
     debug_assert_eq!(z.size(), (1, packed_output_size));
     #[cfg(feature = "debug")]
     if obf_params.hardcoded_key_sigma == 0.0 && obf_params.p_sigma == 0.0 {
-        let hardcoded_key = <<M as PolyMatrix>::P as Poly>::read_from_file(
-            &obf_params.params,
-            &dir_path,
+        let hardcoded_key = timed_read(
```

Comment:
instead of having timed_read and read_from_file as seperate funciton and accept different tag, as we only use timed read only for read from file context, can we combine the logic of computing reading time and updating total_load time that baked into original read_from_file method?

Reply:
yes, only exception is loading `hash_key` which is done using the standard library. But I guess we can consider it as negligible loading time and not count it

Reply:
I see your point but also think that it is better to maintain the logic separated as, for example, `hash_key` is loaded with a function that is different from `read_from_file`

Reply:
Honestly don't think it's trivial to include `hash_key` loading time on total_load time as it's just [u8;32]. I still think having logic combined is better as maintaining two tag string for same variable might cause mistake